pytest/tests/loadtest/contract/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+//! Contract that can be used for different types of loadtesting.
+//! Based on the rust-counter example.
+
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::collections::LookupMap;
+use near_sdk::{env, near_bindgen};
+
+near_sdk::setup_alloc!();
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct Counter {
+    val: i8,
```

Comment:
why `i8`? I don't see where you decrement the value so why not just have a `u64` or `u32`?

Reply:
That was in the original "rust-counter" example. But you're right - changed it to u64